v21.2/restore.md

```diff
@@ -168,6 +169,28 @@ The `RESTORE` process minimizes its impact to the cluster's performance by distr
 When a `RESTORE` fails or is canceled, partially restored data is properly cleaned up. This can have a minor, temporary impact on cluster performance.
 {{site.data.alerts.end}}
 
+## Restoring to multi-region databases
+
+<span class="version-tag">New in v21.2:</span> Restoring to a multi-region database is supported with some limitations. This section outlines details and settings that should be considered when restoring into multi-region databases.
```

Comment:
first mention of 'multi-region database' could link to https://www.cockroachlabs.com/docs/dev/multiregion-overview.html